### PR TITLE
Ft/zenko 255 support centos7

### PR DESCRIPTION
**Pull request by Zempashi**

### CODE REVIEW

File: roles/helm_common/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+bin_dir: /usr/local/bin
```
Comment (NicolasT) [inline roles/helm_common/defaults/main.yml]: This is already set elsewhere, no?

File: roles/check_os/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+- name: 'Assertion on OS'
```
Comment (NicolasT) [inline roles/check_os/tasks/main.yml]: Is there a way to 'override' this assertion?

File: roles/helm_common/defaults/main.yml (hunk 1 above)
Comment (Zempashi) [reply roles/helm_common/defaults/main.yml]: Yes, but considering that:
- bin_dir is misnamed (bin_dir="/usr/bin" ?) and the name of the variable could change
- /usr/local/bin/ is the preferable destination to install helm without package
- we will detect rapidly if helm has been changed of location
- kubespray-defaults is slow

I find reasonnable to have the duplication of the variable

File: roles/check_os/tasks/main.yml (hunk 2 above)
Comment (Zempashi) [reply roles/check_os/tasks/main.yml]: Done

File: roles/helm_common/defaults/main.yml (hunk 1 above)
Comment (Zempashi) [reply roles/helm_common/defaults/main.yml]: But I don't care, so I could change I you want (to be more opensource friendly or whatever)

**Pull request merged**